nattester/nattester.go

```diff
@@ -0,0 +1,105 @@
+// nattester is a tool to quickly test ICE traversal when you have ssh working:
+//
+// scp nattester do.not.leak.hostnames.google.com:
```

Comment:
Might want to change the domain names here? This looks like it might be the result of an open-sourcing tool scrubbing comments?